app/Http/Controllers/Store/RegistrationController.php

```diff
@@ -195,6 +195,18 @@ public function edit(Registration $registration)
         } else {
             abort(404, 'Registration not found.');
         }
+        $evaluations = $registration->centre->sponsor->evaluations;
```

Comment:
This gets the rules we'd append to the defaults

could also be `$registration->getEvaluator()->evaluations` IF it's ok to operate with the combined rules.